model_compression_toolkit/gptq/pytorch/gptq_training.py

```diff
@@ -197,11 +197,12 @@ def gptq_wrapper(self,
         #  quantized, do we need to wrap them as well?
         return layer
 
-    def get_activation_quantizer_holder(self, n: BaseNode) -> Callable:
+    def get_activation_quantizer_holder(self, n: BaseNode, prev_node: BaseNode) -> Callable:
```

Comment:
This is connected to my previous comment,
you need to contract the calls to `get_activation_quantizer_holder` is a way that wouldn't require to pass prev_node in places that it is not require (and to change the signature of unrelated `get_activation_quantizer_holder` functions like this one)